Extensions/IISWebAppDeploy/Tests/Tasks/SqlDacpacDeploy/SqlQueryOnTargetMachines.Tests.ps1

```diff
@@ -4,13 +4,15 @@ $sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path).Replace(".Tests.", ".")
 $VerbosePreference = 'Continue'
 
 $sqlQueryOnTargetMachinesPath = "$currentScriptPath\..\..\..\Src\Tasks\$scriptDirName\TaskModuleSqlUtility\$sut"
+$sqlUtility = "$currentScriptPath\..\..\..\Src\Tasks\$scriptDirName\TaskModuleSqlUtility\sqlUtility.ps1"
```

Comment:
Call this variable jsonConversionUtlity instead of sqlUtility. Comment applies to all relevant places.